walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/personalblog.html

```diff
@@ -1,14 +1,46 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
-    <title>What Makes Me Me?</title>
-    <link rel="stylesheet" href="style.css">
+  <meta charset="UTF-8">
+  <title>What Makes Me Me?</title>
+  <link rel="stylesheet" href="style.css">
   </head>
   <body>
-      <div id="content">
-          <h1>What Makes Me Me?</h1>
-      </div> 
+    <div id="content">
+      <h1>What Makes Me Me?</h1>
+      <img src= "my-images/about-me/blog-jonathan1.jpg"/>
+      <p>
+        My name is Jonathan Aner. I am a rising sophomore Computer Science major attending Hampton University.
+        I am from Philadelphia, the city of brotherly love and home of the world's greatest cheesesteaks. 
+        A suggestion from me is if you don't want to look like a 'tourist' don't go to Pat's or Gino's for 
+        cheesesteaks. I think some of the things that make me who I am today is being raised in Philly, being 
+        surrounded by my friends and family and being a Christian.              
+      </p>
+      <p>
+        Philly is a great city full of history, art and amazing food. I love the city, but that wasn't always
+        the case for me. From a vistor's point of view, you would probably see the city as a somewhat pristine 
+        city with the iconic Rocky Statue and the steps which he climbed from his iconic scene.When you have 
+        lived in Philly, you see the abudance of homelessness and the severity of struggling communities. 
+        I am disappointed this city that people love so much be treated like the trash on street. Now, I find 
+        the beauty in the city because all of the memories I have had here all the way from my youth to know. I
+        would like to see my city's communities improve and the amount homelessness decrease. 
+      </p>
+      <p>
+        My family has been my rock and they have always stood beside me and I truly appreciate them. 
+        They have been a great example of what love should be. I feel truly privelleged to have a family 
+        that loves me even though their beliefs go against my own. They raised me around the love and I 
+        always accepted it. My friends have always been supportive and I have tried my bestdo the same 
+        for them. My friends have been the ones to encourage me to do things that I never thought I could. 
+        Just as my friends encourage me I will always do the same.
+      </p>
+      <img src= "my-images/about-me/blog-jonathan2.jpg"/>
```

Comment:
This photo is larger than the text / other photos. What do you think to use a CSS style to set a max width for photos?

Reply:
I agree, but first I need to move these files because I placed them in the wrong directory. I wanted to merge this pull request and then start editing.